pkg/util/images/mapping.go

```diff
@@ -44,7 +44,7 @@ func loadImageMapFrom(fileName string) (map[string]string, error) {
 
 }
 
-const defaultImageMapFileName = "/etc/operatorImageMap.yaml"
+const defaultImageMapFileName = "operatorImageMap.yaml"
```

Comment:
I am not sure why you need this change? My assumption would be that it breaks the current deployment.

Reply:
In the controller unit tests, it is required to use this map as it will try to lookup the image. It was also convenient when testing it locally not having to set the env.

How about setting the env to the desired location in the Dockerfile? This way we have the location only known in one place.

Reply:
So the argument is to have it more convenient when debugging, rather than having a predictable default when running properly?

For the tests we would manually load this from a different location. You can simple override the loaded map in the unit tests.

Reply:
> So the argument is to have it more convenient when debugging, rather than having a predictable default when running properly?

I think its good practice/not controversial to not have hardcoded paths in the application code, and instead make it part of the deployment configuration, i.e. env variables or command line arguments in the Kubernetes deployment. Ideally I think it should be a 'required' command line/env argument.

Reply:
I would prefer it if the container starts up with sane defaults, without the need to provide additional configuration for the default use case. If that means baking in a an environment variable into the container image than this will work as well.

Grepping through the source code of kubernetes and client-go, I see the pattern of having a bunch of override capabilities (like env vars of command line arguments), and then falling back to a hardcoded value. Which is exactly what we have here as well. But again, my focus is on having a seamless default startup.

Reply:
Updated it to set ENMASSE_IMAGE_MAP_FILE in the Dockerfile